src/components/sortableGridList/sortableGridList.api.json

```diff
@@ -35,12 +35,43 @@
     }
   ],
   "snippet": [
-    "<SortableGridList>",
-    "  data={items$1}",
-    "  maxItemWidth={140$2}",
-    "  itemSpacing={Spacings.s3$3}",
-    "  listPadding={Spacings.s5}",
-    "  onOrderChange={onOrderChange$4}",
-    "/>"
+    "function Example(props) {",
+    "  const data = Assets.data.products.map(product => ({...product}));",
+    "",
+    "  renderItem = ({item}) => {",
+    "    return (",
+    "      <Card flex customValue={item.id}>",
+    "        <Card.Section",
+    "          imageSource={{uri: item.mediaUrl}}",
+    "          imageStyle={{width: '100%', height: 108.7, borderRadius: BorderRadiuses.br10}}",
+    "          imageProps={{",
+    "            customOverlayContent: (",
+    "              <Text margin-s1 h1 orange30>",
+    "                {item.id}",
+    "              </Text>",
+    "            )",
+    "          }}",
+    "        />",
+    "      </Card>",
+    "    );",
+    "  };",
+    "",
+    "  return (",
+    "    <div>",
+    "      <View style={{width: 800}}>",
+    "        <SortableGridList",
+    "          data={data$1}",
+    "          flexMigration",
+    "          // onOrderChange={onOrderChange$2}",
+    "          maxItemWidth={140$3}",
+    "          renderItem={renderItem$4}",
+    "          itemSpacing={Spacings.s3$5}",
+    "          listPadding={Spacings.s5$6}",
+    "          containerWidth={800$7}",
```

Comment:
Let's add the `orderByIndex` prop to the snippet, I think it's important to show